subprojects/ide/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r214/ToolingApiEclipseModelWtpClasspathAttributesCrossVersionSpec.groovy

```diff
@@ -25,7 +24,6 @@ import org.gradle.tooling.model.eclipse.EclipseExternalDependency
 import org.gradle.tooling.model.eclipse.EclipseProject
 import org.gradle.tooling.model.eclipse.EclipseProjectDependency
 
-@TargetGradleVersion('>=2.14')
```

Comment:
Why no `>=3.0` here?

Reply:
Because this is the default in `ToolingApiSpecification` now so there's no need to repeat it here.